Add `The Followpoint: Ephemeral, The Man Behind The Medals` news post (#10338)
ppy/osu-wiki: news/2023/2023-11-03-interview-with-ephemeral.md

---
layout: post
title: "Interview with Ephemeral"
date: 2023-11-03 00:00:00 +0000
---

> Ephemeral, joining the game in August of 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished member of this game.

Reviewer: this feels too long to be a newspost preview line, i'd suggest making a short intro text and move this below the banner

Author: This was decided to just copy from 2013's interview with slight changes, but maybe these days it would be too big, I'm unsure tho

Reviewer: it's really big

![image](https://github.com/ppy/osu-wiki/assets/62819481/d763ec68-6780-41e6-bdf1-2a1c91a41f18)

Author: Okay, yea, that's TOO big

Reviewer: ```suggestion
Come join us delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!
```

Author: Amazing! Only thing is there was a "to" missing

```suggestion
Come join us to delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!
```

> Ephemeral, joining the game in August of 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished member of this game.

Reviewer 2: this intro crucially doesn't mention the reason why eph would be relevant to interview, namely that he makes the medals

Author: That was already solved, so yea

![](meow)

> On October 28th the [osu! Medal Hunters](https://discord.gg/8qpNTs6) [Discord](https://discord.com/) server staff hosted an interview with [Ephemeral](https://osu.ppy.sh/users/102335), and this newspost will highlight best of questions that were asked here! If you wish to view full event recording, you may do so [here](https://youtu.be/OHWbFpPoTJI).

Reviewer: ```suggestion
On October 28th, the [osu! Medal Hunters](https://discord.gg/8qpNTs6) Discord server staff hosted an interview with [Ephemeral](https://osu.ppy.sh/users/102335), and this newspost will highlight some of the best questions that were asked here! If you wish to view the recording of the full event, you may do so [here](https://youtu.be/OHWbFpPoTJI).
```

And so, without further hesitations, let's dive right into it, starting up with a small introduction:

**Who are you?**

I am some guy named Ephemeral, who has been doing osu! things for like, 14 years(?)

**How and why did you join the osu!team?**

> Once upon a time, there was a dickhead named katchup and he made a very bad map, insisting that his map was good. I saw this map and for some reason ketchup's attitude really irked me, so I posted what I guess would be the equivalent of a "mania mod post" in 2009. That kind of got me infamous in the community, because I tore his map to bits, and then I think six months later I was nominated to the [BAT](/wiki/People/Beatmap_Appreciation_Team) at the time, so that was where it all kind of began. 

Author: For maintainers: Yes, that's literally what Ephemeral said

**So how did you start doing medal stuff?**

I recall vaguely that peppy mentioned that he'd like some more achievements, but doesn't know what to do, so I was like: "you know, I got some ideas, I'll send them to you in PM." I sent them to him in PM and said like: "hey, these are pretty good, do you want to do some more?", and I said ok, I'll do some more. 
Then he sent me the spreadsheet they were using for them, and I think I started to kind of tidy it up, because it was all a bit scuffed, as it was just a really early prototype of medals. 
So I think at that point I started getting them in order, and once we had them all in order we ported first generation of medals to the new system, which was called osu!next at the time. The next website started showing the medals properly, and so we started with the Mod Introduction medals and stuff like that. So I've been around involved in medals since then basically.

**What are your thoughts on the current state of medals?**

> Overall I think the past couple of releases, the last release in particular came out fantastic, I'm very happy with it. 

Reviewer 2: what last release?

Author: Here I'd suggest to just link the newspost mentioning it
```suggestion
Overall I think the past couple of releases, [the last release](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in particular came out fantastic, I'm very happy with it. 
```

Author: Applied my suggestion here

> I think they are getting better, but I think with operating at like maybe 20% of what medals will ultimately end up being when they're fully finished being implemented in lazer, because it is very limited what we can do with them at the moment. And lazer will pull back a lot of those limitations and criteria that realistically would not be possible in stable.

Reviewer 2: he's talking about limitations on what things can be checked for medal requirements, but it's assumed that the reader already knows of such things at all

Author: Honestly not sure what to do about this, maybe something like "Writer's note" would work? Feel free to suggest other things you have in mind

Author: Still need to think of a good way to fix this one

**How do you come up with ideas for new medals?**

I stare at a blank wall until they just appear in my head, that's how 90% of them are made.

**Are there any plans for new types of medals?**

We've planned to add multiplayer medals for a long time, and I have so many ideas for those. I will share one briefly, it's more or less a medal that is contagious, you catch it from other people by playing in multiplayer rooms with people that have the medal. I'm really excited for that one!

**Are there any rejected medal ideas that you're willing to share?**

Skylord almost ended up there actually, which some people probably might suggest that it should still be there, you know, because it's really-really hard. 

> There also was a beatmap pack medal that was meant to be for Porter beatmap pack, but I don't recall if we signed Porter. I think we almost had Porter aboard as a Featured Artist, because he was in the same label that features Virtual Self. But I think it fell through and we haven't really pursued it since.

Reviewer 2: "signed"?
porter who?

Author: would that maybe work?

```suggestion
There also was a beatmap pack medal that was meant to be for Porter beatmap pack, but I don't recall if we [signed](/wiki/People/Featured_Artists#selection) [Porter](https://en.wikipedia.org/wiki/Porter_Robinson). I think we almost had Porter aboard as a Featured Artist, because he was in the same label that features Virtual Self. But I think it fell through and we haven't really pursued it since.
```

Reviewer 3: couldn't you just reword it to say "if we signed Porter [Robinson]"? usually people put things between [ ] to elaborate on what people said even if they didn't originally say it when quoting. don't think forcing people to click off onto a wikipedia page to know what you're talking about is a great solution

Reviewer 4: "same label as virtual self" is so wrong i'd probably omit it out of embarrassment.

Author: Done... I think

**What's your favourite medal?**

Probably Internment at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.

**Are there plans for challenge packs in other gamemodes?**

Yep, absolutely. Once we get people willing to tutor us on the maps that would be good for that then yeah.

**Any updates on the next batch of medals?**

There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.

**Any more HalfTime medals?**

> I figured people were tired of those after Aeon, so we haven't really thought about any since.

Reviewer 2: this means nothing to people that don't know the solution to aeon, other than potentially spoiling that it's related to the half time mod

Author: The only suggestion I have for this is just remove it probably, not sure how to make it mean something to them without spoiling Aeon more

Author: Removed this part

**Any more plans for Loved map required medals?**

I wasn't aware until like two packs ago that was even possible, I think someone in here told me that. I thought it was just literally Ranked maps only.

**Do medals track the map you got them on?**

They do, yeah, the whole score is saved, so it's there for us to use later on stuff.

**What's your workout routine?**

My workout routine is sit on a chair like a shrimp for like 18 hours a day.

**Do you plan to make more meme-related medals?**

> If I can put a meme medal in that makes sense - I will at any given opportunity. It's just that you have to be careful when you put those in that you're not forcing the joke. I was worried a bit with 727 that it would be kind of like, you're beating a dead horse, but it's so timeless at this point.

Reviewer: ```suggestion
If I can put a meme medal in that makes sense, I will at any given opportunity. It's just that you have to be careful when you put those in that you're not forcing the joke. I was worried a bit with 727 that it would be kind of like, you're beating a dead horse, but it's so timeless at this point.
```

**Why can't you test medals?**

> I can't test this shit on my account, because the one time I did - you fuckers scraped it immediately, so I can't do that.

Author: Yes, here too

Reviewer 4: Should be censored for the news post.

Author: Done

> **Which medal is worth the most internal score?**

Reviewer 2: what is that

Author: That's the scoring used internally (afaik) to rate how difficult medal is, 1000 points being the hardest, maybe that explanation could also go into "Writer's note" if you agree with this suggestion

Reviewer 4: I've never heard of this, so there's no way readers have. Omit or give more context in the questions, right now this feels shoddy.

Author: I decided to omit it

`Skylord`, `Beast Mode`, `Is This Real Life?` and `Final Boss` are all 1,000.

**Do you think we could do something like an entire playerbase medal, like hit a billion notes in December?**

Yeah, I think so.

**How was Aeon thought of?**

Aeon was inspired by one of my favorite songs, which is also the first beatmap I ever submitted.

**You have such a way with words, and occasionally have us pull out a dictionary to figure out what the medal was even called. Where do you come up with these?**

> I wish I could tell you, but it seems to be a `me` thing, I don't know, I just play a lot of games that involve words like that, and I consume a lot of content. Like, I read fuckloads of books, like fantasy books.

Reviewer: ```suggestion
I wish I could tell you, but it seems to be a *me* thing, I don't know, I just play a lot of games that involve words like that, and I consume a lot of content. Like, I read fuckloads of books, like fantasy books.
```

---

> That's it for the highlighted questions! As a reminder, you can watch a full recording of this interview in the video linked above. Hope you enjoyed it, and... **see you next time**!

Reviewer: ```suggestion
That's it for the highlighted questions! As a reminder, you can watch a full recording of this interview [here](https://youtu.be/OHWbFpPoTJI). Hope you enjoyed it, and... **see you next time**!
```

**Any more HalfTime medals?**

I figured people were tired of those after Aeon, so we haven't really thought about any since.

**Any more plans for Loved map required medals?**

I wasn't aware until like two packs ago that was even possible, I think someone in here told me that. I thought it was just literally Ranked maps only.

**Do medals track the map you got them on?**

They do, yeah, the whole score is saved, so it's there for us to use later on stuff.

**What's your workout routine?**

My workout routine is sit on a chair like a shrimp for like 18 hours a day.

**Do you plan to make more meme-related medals?**

If I can put a meme medal in that makes sense, I will at any given opportunity. It's just that you have to be careful when you put those in that you're not forcing the joke. I was worried a bit with 727 that it would be kind of like, you're beating a dead horse, but it's so timeless at this point.

**Why can't you test medals?**

I can't test this shit on my account, because the one time I did - you fuckers scraped it immediately, so I can't do that.

**Which medal is worth the most internal score?**

> `Skylord`, `Beast Mode`, `Is This Real Life?` and `Final Boss` are all 1,000.

Reviewer 4: Links to medal details on wiki or something?

Author: That would be funny to have, because Hush-Hush medals are kinda... secret you know.  I could link Osekai pages, but you'd murder me for that xD
Anyway's, I removed that question completely

**Do you think we could do something like an entire playerbase medal, like hit a billion notes in December?**

Yeah, I think so.

> **How was Aeon thought of?**

Reviewer 4: What's Aeon? context?

Author: Aeon is one of Hush-Hush medals, not much context to give here imo

---
layout: post
title: "Interview with Ephemeral"
date: 2023-11-03 00:00:00 +0000
---

Come join us to delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!

![](meow)

> Ephemeral, joining the game in August of 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished member of this game.

Author: ```suggestion
Ephemeral, joining the game in August 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished community member.
```

**What are your thoughts on the current state of medals?**

Overall I think the past couple of releases, the [the last release](https://osu.ppy.sh/home/news/2023-07-27-summer-splash) in particular came out fantastic, I'm very happy with it. 
I think they are getting better, but I think with operating at like maybe 20% of what medals will ultimately end up being when they're fully finished being implemented in lazer, because it is very limited what we can do with them at the moment. And lazer will pull back a lot of those limitations and criteria that realistically would not be possible in stable.

**How do you come up with ideas for new medals?**

I stare at a blank wall until they just appear in my head, that's how 90% of them are made.

**Are there any plans for new types of medals?**

We've planned to add multiplayer medals for a long time, and I have so many ideas for those. I will share one briefly, it's more or less a medal that is contagious, you catch it from other people by playing in multiplayer rooms with people that have the medal. I'm really excited for that one!

**Are there any rejected medal ideas that you're willing to share?**

> Skylord almost ended up there actually, which some people probably might suggest that it should still be there, you know, because it's really-really hard. 

Author: seems fair
```suggestion
Skylord almost ended up there actually, which some people probably might suggest that it should still be there, you know, because it's really, really hard.
```

There also was a beatmap pack medal that was meant to be for Porter beatmap pack, but I don't recall if we [signed](/wiki/People/Featured_Artists#selection) [Porter Robinson](https://en.wikipedia.org/wiki/Porter_Robinson). I think we almost had Porter aboard as a Featured Artist, but I think it fell through and we haven't really pursued it since.

**What's your favourite medal?**

> Probably Internment at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.

Reviewer 5: ```suggestion
Probably [Internment](https://osu.ppy.sh/home/news/2023-09-18-the-hunt-for-internment) at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.
```f

> Probably [Internment](https://osu.ppy.sh/home/news/2023-09-18-the-hunt-for-internment) at this point. That was very-very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.

Author: will change it to this:
```suggestion
Probably [Internment](https://osu.ppy.sh/home/news/2023-09-18-the-hunt-for-internment) at this point. That was very, very fun to watch for me to be honest. It's cool when a medal has that much impact, because a lot of the time medals we released people just get in the first 18 hours, and then once the solution is on Osekai they forget about it.
```

**Are there plans for challenge packs in other gamemodes?**

Yep, absolutely. Once we get people willing to tutor us on the maps that would be good for that then yeah.

**Any updates on the next batch of medals?**

There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.

> **Any more plans for Loved map required medals?**

Reviewer 5: ```suggestion
**Any more plans for Loved map required medals? (i.e. Causality and Abrogartion)**
```

> There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.

Reviewer 6: ```suggestion
There may still be some coming out before the end of the year, maybe. They have not been made yet, but we would like to, and it would be a big pack if it were to release and if it does not release this year it'll be early next year. And when I say big pack, I mean like all four modes. Really want to start getting all the modes in on it.
```

I wasn't aware until like two packs ago that was even possible, I think someone in here told me that. I thought it was just literally Ranked maps only.

**Do medals track the map you got them on?**

They do, yeah, the whole score is saved, so it's there for us to use later on stuff.

**What's your workout routine?**

My workout routine is sit on a chair like a shrimp for like 18 hours a day.

**Do you plan to make more meme-related medals?**

If I can put a meme medal in that makes sense, I will at any given opportunity. It's just that you have to be careful when you put those in that you're not forcing the joke. I was worried a bit with 727 that it would be kind of like, you're beating a dead horse, but it's so timeless at this point.

**Why can't you test medals?**

> I can't test this on my account, because the one time I did - you scraped it immediately, so I can't do that.

Reviewer 5: ```suggestion
I can't test this on my account, because the one time I did - [the medal hunters] scraped it immediately, so I can't do that.
```
idk if there needs to be a note of medal hunters scraping developer (Eph and Poon) recent plays and dictionary checks for medal urls 💀

**Do you think we could do something like an entire playerbase medal, like hit a billion notes in December?**

Yeah, I think so.

**How was Aeon (Hush-Hush Medal) thought of?**

> Aeon was inspired by one of my favorite songs, which is also the first beatmap I ever submitted.

Reviewer 5: ```suggestion
Aeon was inspired by [one of my favorite songs](https://osu.ppy.sh/beatmapsets/8000#osu/33485), which is also the first beatmap I ever submitted.
```
